docs/sources/flow/reference/components/otelcol.processor.span.md

```diff
@@ -0,0 +1,379 @@
+---
+title: otelcol.processor.span
```

Comment:
```suggestion
canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.processor.span/
title: otelcol.processor.span
```

Canonical links must be added to metadata block for all new topics. See https://github.com/grafana/agent/pull/4473 for work on existing topics.